src/gui/DatabaseWidget.cpp

```diff
@@ -452,9 +453,21 @@ void DatabaseWidget::openUrl()
 
 void DatabaseWidget::openUrlForEntry(Entry* entry)
 {
-    if (!entry->url().isEmpty()) {
-        QDesktopServices::openUrl(entry->url());
+    QString UrlString = entry->url();
+    if (UrlString.isEmpty())
+        return;
+
+    UrlString.replace("{TITLE}", entry->title(), Qt::CaseInsensitive);
```

Comment:
- Please create atomic commits, i.e. one change per commit.
- There is already Entry::resolvePlaceholders()